must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -116,6 +116,12 @@ for ns in $namespaces; do
              pids_ceph+=($!)
              { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "${ceph_commands[$i]} --connect-timeout=15 --format json-pretty" >> "${JSON_COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-"${ceph_commands[$i]}"-json-debug.log 2>&1 &
              pids_ceph+=($!)
+
+             if (( i % 20 == 0 ))
+             then
+               printf "waiting 10s for PIDs to be empty before proceeding \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+               sleep 10
+             fi
```

Comment:
Can you explain what you mean by "limit in PIDs"? https://bugzilla.redhat.com/show_bug.cgi?id=1956285

While running must-gather I saw that value of `ps aux | wc -l` goes till 115 maximum and it fails to collect the resources when it's above 115, so to keep a buffer maybe we could take it to be 100 and then have something like:


```suggestion
process_running=$(ps aux | wc -l)
if (( process_running -gt 100 ))
             then
               printf "waiting 10s for PIDs to be empty before proceeding \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
               sleep 10
             fi`
```

Reply:
Oh, I see... so we're hitting a weird CRI-O limitation. While your proposal is not ideal, I don't think there's any easy way for us to determine the full number of running processes on the node itself, so this is the best we can do. I'd like to propose something a little different though: Instead of an `if` do a `while`. While the number of running processes is >100, wait for either 2 or 5 seconds, then continue once the count is less than 100.

In any case, please also add some comments above these new lines indicating 1. why we're doing the wait and 2. why we chose 100. Specifically mention the CRI-O and link this article: https://access.redhat.com/solutions/5597061 Both of these things will be crucial to know in the future.